pandas/sparse/tests/test_frame.py

```diff
@@ -879,7 +892,15 @@ def test_numpy_cumsum(self):
         tm.assertRaisesRegexp(ValueError, msg, np.cumsum,
                               self.frame, out=result)
 
+    def test_numpy_func_call(self):
+        # no exception should be raised even though
+        # numpy passes in 'axis=None'
+        funcs = ['sum', 'cumsum', 'var', 'mean', 'prod',
+                 'cumprod', 'std']
```

Comment:
`np.kurt`, `np.skew`, `np.sem`, `np.idxmin`, and `np.idxmax` do not exist.  Will add for `np.max` and `np.min` though.